library/Icinga/Web/Response/JsonResponse.php

```diff
@@ -190,7 +222,9 @@ public function outputBody()
                 $body['data'] = $this->getSuccessData();
                 break;
         }
-        echo json_encode($body, $this->getEncodingOptions());
+        echo $this->autoSanitize
```

Comment:
Please use `getAutoSanitize()`. Subclasses may override this method. Though we don't any at moment.

Reply:
Always used `$this->whatever` – no one complained. Shall I do member reads like you everywhere in the future?

Reply:
I stumbled upon this several times while working on the ipl. I'd say that we did not care about subclass overrides before. Yes, please use the getter for reads in the future.